docs/getting-started/building-instui.md

```diff
@@ -0,0 +1,59 @@
+---
+title: How to build Instructure UI
+category: Getting Started
+order: 4
+---
+
+# Building Instructure UI
+
+Instructure UI is a mono-repository which means it has multiple packages (almost always a component means an individual package) and they are treated as one whole package when dealing with dependencies and building them.
+
+The project uses `yarn` for managing these packages and also for resolving dependencies, running scripts and managing the whole mono-repository itself.
+
+> **Note**: Windows OS is not supported as a build platform due to some libraries that we rely on for building Instructure UI do not support Windows
```

Comment:
Also we're using environment variables through our build pipeline and they are defined differently in Linux/OSX and Windows.